help/destinations/catalog/crm/salesforce.md

```diff
@@ -0,0 +1,161 @@
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Custom Domain]**: Your Salesforce domain.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+## Mapping Example {#mapping-example}
+
+In the select target window, When selecting the target field choose the **Select custom attributes** category and add the mappings' desired.
+
+For instance, your XDM profile schema and your [!DNL Salesforce CRM] instance could have the attribute mapping as below:
+
+||XDM Profile Schema|[!DNL Salesforce CRM] Instance| Mandatory|
+|---|---|---|---|
+|Attributes|<ul><li><code>person.name.firstName</code></li><li><code>person.name.lastName</code></li><li><code>personalEmail.address</code></li></ul>|<ul><li><code>FirstName</code></li><li><code>LastName</code></li><li><code>Email</code></li></ul>|
+|Identities|<ul><li><code>crmID</code></li></ul>|<ul><li><code>SalesforceId</code></li></ul>|Yes|
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** to navigate to the list of destinations.
+![Browse Destinations](../../assets/catalog/crm/salesforce/browse-destinations.png)
+
+1. Select the destination and validate that the status is **[!UICONTROL enabled]**.
+![Destinations Dataflow Run](../../assets/catalog/crm/salesforce/destination-dataflow-run.png)
+
+1. Switch to the **[!DNL Activation data]** tab, then select a segment name.
+![Destinations Activation Data](../../assets/catalog/crm/salesforce/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count created within the segment.
+![Segment](../../assets/catalog/crm/salesforce/segment.png)
+
+1. Login to the Salesforce website then navigate to the **[!DNL Apps]** > **[!DNL Contacts]** page and check if the profiles from the segment have been added.
+![Salesforce Contacts](../../assets/catalog/crm/salesforce/contacts.png)
+
+1. Click a contact and check if the attribute is updated. You will notice the segment id's which were used to update against the `AdobeExperiencePlatformSegments` custom field.
+![Salesforce Contacts](../../assets/catalog/crm/salesforce/contact-info.png)
```

Comment:
This image has a lot of unnecessary whitespace at the bottom. Please edit to remove the whitespace.